unime/src/lib/components/image/Image.svelte

```diff
@@ -20,6 +20,9 @@
   export let isTempAsset = false;
   let assetUrl: string | null = null;
 
+  export let useFallback: boolean = false;
+  $: useFallback = assetUrl === null;
```

Comment:
I would add a comment here on why this needs to be reactive. I guess the component mounts with `assetUrl=null` and only then `onMount` runs and updates it?